### DISCUSSION TITLE
Automatic LOD generation

### DISCUSSION THREAD

**Discussion by L4exp** [o3de/o3de#9742] [Ideas]
There is one cool feature which can save a lot of game dev time.
It is Automatic LOD Generation.
We have game ready mesh with pretty good topology and textures. Textures can be in UDIM scheme.
It very helpful feature to generate N LODs of mesh for different distances by AUTOMATIC way with various strategies of decreasing polycount(percentage, formula, static values) for each LOD.
Maybe, it makes sence to do this on prefab level where also textures(UDIM or Singe tiles which used on mesh) can be generated to lower resolution too, by necessity

**Comment by rossbridger** [comment]
FYI, my PR https://github.com/o3de/o3de/pull/18711 implemented this feature.
